docs/CONTRIBUTING.md

```diff
@@ -112,6 +116,37 @@ In our GitHub Actions CI, we use CentOS 7 for maximum compatibility.
 If you need your builds to support older distros, run the build commands
 inside a Docker container with all the build requirements installed.
 
+## Testing
+
+There are three kinds of tests in code-server:
+
+1. unit tests
+2. integration tests
+3. end-to-end tests
+
+### Unit Tests
+
+Our unit tests are written in TypeScript and run using [Jest](https://jestjs.io/).
```

Comment:
Is there a short explanation of what Jest is that we could add here, maybe like "Jest testing framework"? Is Jest used for both unit and integration tests, or just intended for integration tests?

Reply:
Really good point!